wellcomeml/ml/cnn.py

```diff
@@ -49,6 +52,12 @@ def residual_conv_block(x1):
             x2 = tf.keras.layers.LayerNormalization()(x2)
             return tf.keras.layers.add([x1, x2])
 
+        def residual_attention(x1):
+            x2 = HierarchicalAttention()(x1)
```

Comment:
How come you initialise this but you don't do `build` or `call`?

Reply:
It happens under the hood from tensorflow

Reply:
https://www.tensorflow.org/guide/keras/custom_layers_and_models

Reply:
oo nice